lib/Reader.php

```diff
@@ -22,6 +22,12 @@ class Reader extends XMLReader {
     use ContextStackTrait;
 
     /**
+     * Reading indicator
+     * @var bool
+     */
+    public $reading = false;
```

Comment:
Changed it to protected in https://github.com/fruux/sabre-xml/pull/110/commits/548fc32d613e374429d13fe6172b7fa408a50893.